### PR TITLE
added matrix_power function

### PR DESCRIPTION
**Pull request by ndurumo254**
hello, have added a matrix_power function  in the front end jax.

### CODE REVIEW

**Comment by ndurumo254** [comment]
hello, once i try to run the test , am getting the era that i skipped the process.  how am i supposed to carry the test?

File: ivy/functional/frontends/jax/lax/linalg.py [hunk 1]
```diff
@@ -1,4 +1,8 @@
 import ivy
+from jax.numpy import linalg
+
+from brainpy.math.jaxarray import JaxArray
+from brainpy.math.numpy_ops import _remove_jaxarray
```
Comment (jkeane508) [inline ivy/functional/frontends/jax/lax/linalg.py:5]: same here

File: .idea/misc.xml [hunk 2]
```diff
@@ -3,8 +3,8 @@
   <component name="JavaScriptSettings">
     <option name="languageLevel" value="ES6" />
   </component>
-  <component name="ProjectRootManager" version="2" project-jdk-name="Remote Python 3.8.10 Docker (unifyai/ivy:latest)" project-jdk-type="Python SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.10" project-jdk-type="Python SDK" />
```
Comment (jkeane508) [inline .idea/misc.xml:6]: All changes in this file need to be undone

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Hi I think you should refer to the [docs](https://lets-unify.ai/ivy/deep_dive/16_ivy_frontends.html) to see how to do the correct frontend implementations

I have highlighted the changes necessary :)

Can you also reference the issue you created for this task so I can link it to this PR.

**Comment by ndurumo254** [comment]
am trying to figure out how I can reference the issue after I created it. would someone help me?

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Hey,

You still need to make the changes I outlined in the previous review :)

Moreover, there seems to be some lint failures, you can import a pre commit hook here which will format code automatically before every commit [here](https://lets-unify.ai/ivy/contributing/0_setting_up.html#pre-commit)

**Review by ndurumo254 — COMMENTED** [review]
am getting challenges with pytest library. how can i use it. i have installed it yet i get error that the module is not found

**Review by ndurumo254 — COMMENTED** [review]
am getting challenges with pytest library. how can i use it. i have installed it yet i get error that the module is not found

**Comment by jkeane508** [comment]
Hi,

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Hi,

It seems that you have made changes to a lot of files on your fork. Unfortunately for PRs we only accept changes to files that are relevant to the task you are working on. Before I can merge this you need to remove changes to files that are not relevant :)

**Pull request closed without merging**